.env.example

```
@@ -14,3 +14,25 @@ AWS_REGION=ap-northeast-2
+JWT_REFRESH_TOKEN_DAYS=7
+JWT_REFRESH_TOKEN_REMEMBER_DAYS=30
+AUTH_CONSENT_VERSION=2026-04-v1
+
+# Email verification policy
+AUTH_EMAIL_STORE=redis
+AUTH_EMAIL_CODE_TTL_SECONDS=300
+AUTH_EMAIL_RESEND_COOLDOWN_SECONDS=60
+AUTH_EMAIL_MAX_ATTEMPTS=5
+AUTH_EMAIL_VERIFIED_TTL_SECONDS=1800
+AUTH_EMAIL_FROM_ADDRESS=
+AUTH_EMAIL_SUBJECT=[GACHI] Email verification code
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_EMAIL_SUBJECT key should go before the AUTH_EMAIL_VERIFIED_TTL_SECONDS key

(UnorderedKey)